vignettes/Introduction_agestructEPICYST.Rmd

```diff
@@ -0,0 +1,164 @@
+
+For humans in the default model, there are 7 age groups, representing 0 - 4.99, 5 - 9.99, 10 - 14.99, 15 - 29.99, 30 - 49.99, 50 - 69.99 and 70 - 89.99 years. These age groups are structured to include school-age children (5 - 9.99 & 10 - 14.99 yrs). While there is an input to change these `set_up(number_age_classes_human = 7)`, leaving this default model for human age-structuring is recommended. 
+
+Figure 1 below provides a representation of the age-structure in the model, including demographic and transmission parameters. 
+
+```{r echo=FALSE, out.width='100%'}
+knitr::include_graphics('./EPICYST_schematic_legend.jpg')
+```
+
+## Other key input parameters in the age-structured pig model
+
+Input parameters into `set_up()` specify additional age-specific features in the pig age-structured model. Firstly, the `set_up(slaughter_age_min = 6)` input
+specifies the minimum age (in months) in which pigs are slaughtered, which is set to 6 months in the default model. Secondly, the `set_up(slgEP = 1)` specifies the number of years pigs are expected live, on average, after reaching slaughter age (a default for the average natural life expectancy for pigs is also specified by `set_up(LEP = 10)`). 
+
+Figure 2 below highlights how pig demographics have been incorporated into the age-structured EPICYST, which simyulates pigs before the minimum slaughter age experiencing a constant natural mortality rate (dP; the reciprocal of the natural life expectancy parameter, `LEP`), and pigs after reaching the minimum slaughter age experiencing an additional mortality rate associated with reduced survival due to slaughter (dPslg; the reciprocal of the average life expectancy after reaching slaughter age, `slgEP`). Equally, humans are only exposed to infected pork from slaughter age animals. 
+
+```{r echo=FALSE, out.width='100%'}
+knitr::include_graphics('./Pig_demographic_legendFig2.jpg')
+```
+Other modifiable parameter inputs, impacting baseline demographic parameters in `set_up()`, human population size `HPS`, pig population size `PPS`, human life expectancy (in years) `LEH`, and the rate of pork meal procurement in humans `chi`.  
+
+EPICYST can also be parameterised for specific epidemiological conditions by specifying epidemiological parameters in `set_up()`, including Egg to pig transmission coefficient `tau_input`, Egg to pig transmission coefficient `theta_input`, Pig to human transmission coefficient `beta_input`, and the proportion of pigs with low-cyst burden `phi`.
+
+# Run the age-structured model with no interventions
+
+The `run_model()` function provides all functionality for model runs. To run the model at baseline, with default parameters and endemic equilibrium values, only the length of the model run (specified in years) need be specified:
+
+```{r}
+m1 <- run_model(time = 50)
```

Comment:
Running this first example we seem to be "leaking" pigs - this should be fixed (and probably tested for).

```
# Where are the pigs going?
pigs <- rowSums((m1[, c(33:932)]))
pigs[c(1, nrow(m1))]

m1[c(1, nrow(m1)), "Pig_total"]
```

Reply:
Error now fixed and will be included in the updated PR